src/pages/Encounters/tabs/EncounterOverviewTab.tsx

```diff
@@ -58,8 +58,8 @@ export const EncounterOverviewTab = ({
         {/* Left Column - Symptoms, Diagnoses, and Questionnaire Responses */}
         <div className="flex-1 space-y-4" data-cy="encounter-overview">
           {canEdit && (
-            <div className="flex justify-between gap-2">
-              <div className="grid grid-cols-2 sm:grid-cols-3 gap-2 mx-auto md:mx-0">
+            <div className="flex flex-row justify-between gap-2">
+              <div className="grid grid-cols-2 sm:grid-cols-3  sm:w-auto gap-2 justify-start">
```

Comment:
Why use grid when the same layout could be achieved with flex and flex-wrap? If you check, the third button moves to the first column instead of continuing in the row, even when there appears to be enough space. Also, why use `sm:w-auto `? Isn’t w-auto the default behaviour?

```suggestion
              <div className="flex flex-wrap gap-2 justify-start">
```